Documentation/Stage-1-Introduction/Day-1/Index.rst

```diff
@@ -83,6 +82,6 @@ Practical goals
 
 By the end of this day, you should have completed the following:
 
-* Item 1
-* Item 2
-* Item 3
+* Practical prerequisites are verified
+* Theoretical prerequisites are verified
+* Theoretical goals of the day are achieved
```

Comment:
Write these out as completed tasks. So, how do we practically give basic understanding of the things mentioned in the theoretical prerequisites?